packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1558,6 +1561,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     ): void {
         this.verifyNotClosed();
 
+        if (this.deltaManager.storageOnly) {
```

Comment:
Can we not add this code? It will fail one way or another (nack on submitting op, which should return non-retriable error), why add more tracking here?

Reply:
currently there will be no nack, as ops are not submitted anywhere, as there is nowhere to submit them to